src/transport/async_channel.rs

```diff
@@ -0,0 +1,364 @@
+//! Memory transport implementation using [tokio::sync::mpsc]
```

Comment:
So this was originally intended to use the async_channel crate? Otherwise the name is confusing.

Reply:
yes, see todo to rename

Reply:
renamed